R/summary_functions.R

```diff
@@ -4,8 +4,9 @@
 #' @export
 percent_coverage <- function(coverage_result){
   coverage_data <- as.data.frame(coverage_result)
-  num_lines <- nrow(coverage_data)
-  num_not_zero <- sum(coverage_data$value != 0) # how many lines have a value that is not zero?
+  num_lines <- length(unique(coverage_data$first_line))
```

Comment:
As I wrote in the introduction, the sum(coverage_result$value > 0) / NROW(coverage_result) is wrong because it assumes that all first_line are unique. I provided a test that shows it.